Source/CesiumRuntime/Public/CesiumRasterOverlay.h

```diff
@@ -125,8 +137,33 @@ class CESIUMRUNTIME_API UCesiumRasterOverlay : public UActorComponent {
   UFUNCTION(BlueprintCallable, Category = "Cesium")
   void SetSubTileCacheBytes(int64 Value);
 
+  /**
+   * Activates this raster overlay, which will add it to (that is: display it
+   * on) the Cesium3DTileset to which the component is attached, if it hasn't
+   * already been added. The overlay will continue to be shown on the tileset
+   * until it is deactivated.
```

Comment:
Maybe this is a personal preference, but I think flipping the verbiage here relates this to `AddToTileset` a bit more clearly

```suggestion
   * Activates this raster overlay, which will display it on (that is: add it to)
   * the Cesium3DTileset to which the component is attached, if it isn't
   * already displayed. The overlay will continue to be shown on the
   * tileset until it is deactivated.
```